### DISCUSSION TITLE
Where can I find the source code for `libnut`?

### DISCUSSION THREAD

**Discussion by dscho** [nut-tree/nut.js#471] [Q&A] [answered]
In the past, I have been contributing fixes to `libnut` but now I cannot find the repository where my PRs were merged... Help?

**Comment by s1hofmann** [comment, marked as answer]
I did some restructuring while working on v3.0.0 and changed its visibility while I thought about some things

It’s now here: https://github.com/nut-tree/libnut-core

**Reply by dscho** [reply]
Thank you for resurrecting it to public.

Unfortunately, it does not reconnect my fork into the same network 😞 
![image](https://user-images.githubusercontent.com/127790/214268912-137dbb57-fa51-44fd-af5c-eaf4e6df03ad.png)
